services/mailer/mailer.go

```diff
@@ -67,6 +70,10 @@ func (m *Message) ToMessage() *gomail.Message {
 		msg.SetBody("text/plain", plainBody)
 		msg.AddAlternative("text/html", m.Body)
 	}
+
+	if len(msg.GetHeader("Message-ID")) == 0 {
+		msg.SetHeader("Message-ID", "<autogen-"+m.generateMessageID()+">")
```

Comment:
The result is `<autogen-<...@...>>` which does not look correct.

Reply:
oh, you're right